src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    setStatus: (state, action) => {
+      state.status = action.payload;
+    },
```

Comment:
- [ ] Great work so far! However, according to the project requirements, the reducer inside the categories slice needs to check the status before updating the state. It appears that this step has been missed in your code, so please ensure that you check the payload from the action before making any updates to the state, Here is an example of how to do this:
```jsx
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories

```
- [ ] As you might've guessed this also means you only need one state (categories) in this slice and it's that state that should be updated so please remove the `status` property from the initial state.